### DISCUSSION TITLE
0.16 lpr and face

### DISCUSSION THREAD

**Discussion by divemasterjm** [blakeblackshear/frigate#14607] [General]
I'm very interested trying the new features in 0.16, is it possible? in the 0.16 repo i cant see any package.
i can make test for you if you like.

Thanks in advance

**Comment by hawkeye217** [comment]
These features are still very much in active planning and development and won't be available in a package for a while. Thanks for your interest, though.

**Reply by damianssj661** [reply]
Do you have any estimated time for 0.16? We talk about 1-2 months or a year?
Thanks

**Reply by NickM-27** [reply]
0.15 is still in beta and the full scope of 0.16 is not defined yet, there is no estimate
